### PR TITLE
UX enhancement for the proctoring instructions page.

### PR DESCRIPTION
**Pull request by idegtiarov**
**Description:**
Recently, other proctoring providers could have onboarding without
software download step, though `Start System Check` exists on the
page and confusing students.

Currently, the button for software downloading covered by conditions.

**Pre-Merge Checklist:**

- [x] Described your changes in `CHANGELOG.rst`
- [x] Confirmed Github reports all automated tests/checks are passing.
- [x] Approved by at least one additional reviewer.

**Post-Merge:**

- [ ] Create a tag matching the new version number.

### CODE REVIEW

**Comment by edx-webhook** [comment]
Thanks for the pull request, @idegtiarov! I've created [OSPR-5247](https://openedx.atlassian.net/browse/OSPR-5247) to keep track of it in JIRA.

As a core committer in this repo, you can merge this once the pull request is approved per the [core committer reviewer requirements](https://openedx.atlassian.net/wiki/spaces/COMM/pages/1529675973/Open+edX+Core+Committers#Reviews-and-Ownership) and according to the agreement with your edX Champion.

**Comment by natabene** [comment]
@idegtiarov Thank you for your contribution.

**Comment by natabene** [comment]
@marcotuts Do you want to review this before it goes to engineering?

**Comment by natabene** [comment]
@idegtiarov Could you please provide more information and detail on what this PR is trying to achieve or the problem is it solving, so our product manager can review?

**Comment by idegtiarov** [comment]
Hi @natabene thanks for your attention and question to the PR.

The main problem that is solved relates to the onboarding proctoring process that is different from other proctored providers.
Since proctored provider could be added to the platform as an installable application that requires pretty straightforward steps: install provider backend, add proctoring settings. 

Some settings variable could be redundant, in the solved case this is `software_download_url` and as a result button on the instructions page `Start System Check` leads nowhere. The proposal is to cover DOM element by conditional case and not rendering an odd element if it is not valid one.

**Comment by natabene** [comment]
@idegtiarov Thank you for providing additional information, we will look into it.

**Review by Zacharis278 — APPROVED** [review]
👍

**Comment by edx-webhook** [comment]
@idegtiarov 🎉 Your pull request was merged!

Please take a moment to [answer a two question survey](https://docs.google.com/forms/d/e/1FAIpQLSceJOyGJ6JOzfy6lyR3T7EW_71OWUnNQXp68Fymsk3MkNoSDg/viewform?usp=pp_url&entry.1671973413=edx/edx-proctoring&entry.867055334=https://github.com/edx/edx-proctoring/pull/723&entry.1484655318=https://github.com/idegtiarov&entry.752974735=2020-11-24+12:46&entry.1917517419=2020-12-17+21:19&entry.2133058324=Yes) so we can improve your experience in the future.

**Comment by edx-webhook** [comment]
@nedbat, @nasthagiri, @schenedx: thought you might like to know that idegtiarov merged this pull request.

**Pull request merged**
